src/libraries/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -98,7 +98,6 @@
     <EmbeddedResource Include="Resources\Interop\OtherAccessors.dll" />
     <EmbeddedResource Include="Resources\Interop\Interop.Mock01.dll" />
     <EmbeddedResource Include="Resources\Interop\Interop.Mock01.Impl.dll" />
-    <None Include="Resources\Misc\CPPClassLibrary2.h" />
```

Comment:
CPPClassLibrary2.obj is generated from this file. It should stay.

Reply:
Is the object file also unused by metadata tests (since there is no assertion failure with/without this header)?

Reply:
It is there for reference of what `CPPClassLibrary2.obj` was built from. https://github.com/dotnet/runtime/blob/master/src/libraries/System.Reflection.Metadata/tests/Resources/README.md#building-the-binary-files talks about it.